00-Raster-Structure.Rmd

```diff
@@ -225,54 +228,53 @@ plot(DSM_HARV,
     differences are a direct result of the calculations used to "flatten" the 
-    data onto a 2 dimensional map. Source: opennews.org</figcaption>
+    data onto a 2 dimensional map. Image source: opennews.org</figcaption>
 </figure>
 
 ##What Makes Spatial Data Line Up On A Map?
-There are lots of great resources that describe coordinate reference systems and 
-projections in greater detail. However, for the purposes of this activity, what 
+There are lots of great resources that describe coordinate reference systems and
+projections in greater detail. For the purposes of this activity, what 
 is important to understand is that data from the same location but saved in 
 different projections **will not line up in any GIS or other program**. Thus 
 it's important when working with spatial data in a program like `R` or `Python` 
-to identify the coordinate reference system applied to the data, and to grab 
-that information and retain it when you process / analyze the data.
+to identify the coordinate reference system applied to the data and retain it 
+throughout the processing and analysing of the data.
 
 Read More: 
-* <a href="http://spatialreference.org/ref/epsg/" target="_blank">A comprehensive
+
+* <a href="http://spatialreference.org/ref/epsg/" target="_blank"> A comprehensive
 online library of CRS information.</a>
-* <a href="http://docs.qgis.org/2.0/en/docs/gentle_gis_introduction/coordinate_reference_systems.html" target="_blank">QGIS Documentation - CRS  Overview</a>
-<a href="https://source.opennews.org/en-US/learning/choosing-right-map-projection/" target="_blank">Choosing the Right Map Projection.</a>
+* <a href="http://docs.qgis.org/2.0/en/docs/gentle_gis_introduction/coordinate_reference_systems.html" target="_blank">QGIS Documentation - CRS Overview</a>
+* <a href="https://source.opennews.org/en-US/learning/choosing-right-map-projection/" target="_blank">Choosing the Right Map Projection.</a>
 * <a href="https://www.nceas.ucsb.edu/~frazier/RSpatialGuides/OverviewCoordinateReferenceSystems.pdf" target="_blank"> NCEAS Overview of CRS in R</a>
 
-####How Map Projections Can Fool the Eye
+###How Map Projections Can Fool the Eye
 Check out this short video highlighting how map projections can make continents 
 seems proportionally larger or smaller than they actually are!
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/KUF_Ckv8HbE" frameborder="0" allowfullscreen></iframe>
 
-
-
 While we will not go into great depth with respect to understanding Coordinate
-reference systems in this lesson, it is important to understand that 
-if your data are in different CRS', then they will not line up in R. They might
-line up in a GIS interface that knows how to "reproject on the fly" to make things
-visually line up. 
+Reference Systems in this lesson, it is important to understand that 
+if your data are in different CRSs, then they will not line up in `R`. (They may
```

Comment:
I used the code format when it was applying to the R entity but not if it was a concept in general.  A bit confusing when the terms are the same, but it applies easily when the code term is different or only a small part of the full term.   Here we are thinking about CRSs in general not the CRS in a particular R object.  Is that too confusing?